backend/scripts/sweepstakes-email.ts

```diff
@@ -0,0 +1,127 @@
+import { runScript } from 'run-script'
+import fetch from 'node-fetch'
+
+const MAILGUN_API_KEY = '' // add API key
+
+const TEST_SEND_TO_SELF = process.argv.includes('--test-self') // Add flag to script to send email to yourself to test
+const TEST_ONLY_FETCH = process.argv.includes('--test-fetch') // Add flag to script to fetch number of users but don’t send emails
+
+const MY_USER_ID = '' // Update for --test-self email
+
+type SweepstakesUser = {
+  id: string
+  username: string
+  email: string
+}
+
+async function getSweepstakesVerifiedUsers(
+  pg: any
+): Promise<SweepstakesUser[]> {
+  const users: SweepstakesUser[] = await pg.manyOrNone(
+    `
+    SELECT 
+      u.id, 
+      u.username, 
+      pu.data ->> 'email' AS email
+    FROM users u
+    JOIN private_users pu ON u.id = pu.id
+    WHERE u.data ->> 'sweepstakesVerified' = 'true'
+    `
+  )
+
+  return users.filter((user) => user.email !== null)
+}
+
+async function getMyUser(pg: any): Promise<SweepstakesUser | null> {
+  const user: SweepstakesUser | null = await pg.oneOrNone(
+    `
+    SELECT 
+      u.id, 
+      u.username, 
+      pu.data ->> 'email' AS email
+    FROM users u
+    JOIN private_users pu ON u.id = pu.id
+    WHERE u.id = $1
+    `,
+    [MY_USER_ID]
+  )
+
+  return user && user.email ? user : null
+}
+
+async function sendBulkEmails(users: SweepstakesUser[]) {
+  if (users.length === 0) {
+    console.log('No users to send emails to.')
+    return
+  }
+
+  const mailgunUrl = `https://api.mailgun.net/v3/mg.manifold.markets/messages`
+
+  const recipientData = users.reduce((acc, user) => {
+    acc[user.email] = { username: user.username }
+    return acc
+  }, {} as Record<string, { username: string }>)
+
+  const emailParams = new URLSearchParams()
+  emailParams.append('from', 'Manifold <info@manifold.markets>')
+  emailParams.append('to', users.map((u) => u.email).join(','))
```

Comment:
I don't think this is how you do this. Why don't you use the `sendBulkEmails` function in `backend/shares/src/emails.ts` that will handle the 'to' fields. The way that function handles the 'to' data is each a separate field, whereas yours looks like everyone will see everyone else's email, which would be very bad.